ui/jest.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     moduleDirectories: ['node_modules'],
     reporters: ['default', 'jest-junit'],
     setupFilesAfterEnv: [
-        '@testing-library/jest-dom/extend-expect',
+        '@testing-library/jest-dom',
```

Comment:
jest-dom major version upgrade breaking change https://github.com/testing-library/jest-dom/releases/tag/v6.0.0 
> extend-expect was not documented in the Readme, so this change should
have minimal impact.
> Users can now use the following import paths to automatically extend
"expect" for their chosen test platform:
> @testing-library/jest-dom - jest (@types/jest)